app-hw-37/src/FullScreenCard/FullScreenCard.tsx

```diff
@@ -1,15 +1,26 @@
-import React, { ReactNode } from 'react';
-import { useState, useEffect } from 'react';
-import { getPost } from '../getPosts';
 import './fullScreenCard.css';
 import { Link, useParams } from 'react-router-dom';
-import { useAppSelector } from '../Store/store';
+import { useAppDispatch, useAppSelector } from '../Store/store';
+import { Likes } from '../Likes/Likes';
+import { Mark } from '../CardsList/Card';
+import { removeFromFavourites, setFavourites } from '../Store/favorites';
+import { setPostsIds } from '../Store/post';
 
 export const FullScreenCard = () => {
 	const params = useParams()
+	const dispatch = useAppDispatch()
+
 	const post = useAppSelector((state) => state.posts.posts.find((post) => String(post.id) === params.postId))
 	const theme = useAppSelector((state) => state.theme.themeColor)
-
+	const favourites = useAppSelector((state) => state.favsPosts.favPosts)
+	const favourite = favourites.find((postCard) => {
```

Comment:
you can do it inside Selector, it will add memorization.
But honestly - it not a big deal)